### PR TITLE
Version v0.2 - Introduce scientific benchmarks and code refactoring

### PR DESCRIPTION
**Pull request by vedran-kasalica**

### CODE REVIEW

File: src/workflomics_benchmarker/cwl_utils.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+            The path to the cwl workflow file.
+            
+        Returns
+        -------
+        List[str]
+            The list of step names.
+        """
+        with open(workflow_file, 'r') as file:
+            data = yaml.safe_load(file)
+        steps = []
```
Comment (kretep) [inline src/workflomics_benchmarker/cwl_utils.py]: Just a detail obviously, but more idiomatic for Python would be `return [step_name for step_name in data.get('steps', {})]`

File: src/workflomics_benchmarker/scientific_benchmarks.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+
+
+def calculate_output_file(dir_path, workflow_name, output_name):
+    """
+    Calculate the path to the output file for a given workflow.
+    """
+    return os.path.join(dir_path, workflow_name + "_output", output_name)
+
+# dir_path = "/Users/vedran/Library/CloudStorage/OneDrive-NetherlandseScienceCenter/eScience/All Projects/Workflomics/Events/IEEE eScience/runs/demo_gProfiler_big"
```
Comment (kretep) [inline src/workflomics_benchmarker/scientific_benchmarks.py]: I'd recommend either removing the commented-out code or put a note here why you want to keep it

**Review by kretep — APPROVED** [review]
See some minor comments, but otherwise seems fine

File: src/workflomics_benchmarker/cwl_utils.py (hunk 1 above)
Comment (vedran-kasalica) [reply src/workflomics_benchmarker/cwl_utils.py]: I updated it, thanks!

File: src/workflomics_benchmarker/scientific_benchmarks.py (hunk 2 above)
Comment (vedran-kasalica) [reply src/workflomics_benchmarker/scientific_benchmarks.py]: Good point, I removed the commented-out code

**Pull request merged**